packages/hash/api/src/graphql/resolvers/knowledge/page/page.ts

```diff
@@ -1,7 +1,12 @@
-import { PageModel } from "../../../../model";
+import { PageModel, UserModel } from "../../../../model";
 
-import { QueryKnowledgePageArgs, ResolverFn } from "../../../apiTypes.gen";
-import { GraphQLContext } from "../../../context";
+import {
+  MutationCreateKnowledgePageArgs,
+  QueryKnowledgePageArgs,
+  QueryKnowledgePagesArgs,
+  ResolverFn,
+} from "../../../apiTypes.gen";
+import { GraphQLContext, LoggedInGraphQLContext } from "../../../context";
 import {
   UnresolvedKnowledgePageGQL,
   mapPageModelToGQL,
```

Comment:
Unrelated to your PR but something I'd like us to clean up at some point.. the inconsistencies of PascalCase with acronyms is quite frustrating to me. (In some other types we'd have called this `mapPageModelToGql`)

Reply:
I agree with this, and think we should do a big case-sensitive find and replace post-merge (there's currently 407 incorrect uses)